AppDB/groomer.py

```diff
@@ -980,7 +1377,11 @@ def main():
   logging.debug("Trying to get groomer lock.")
   if ds_groomer.get_groomer_lock():
     logging.info("Got the groomer lock.")
-    ds_groomer.run_groomer()
+    try:
```

Comment:
My guess is that we'd want to do that at the process invocation level so that we have the option to run it normally if we want to.